compiler-core/src/type_/tests/snapshots/gleam_core__type___tests__imports__import_value_as_submodule.snap

```diff
@@ -0,0 +1,14 @@
+---
+source: compiler-core/src/type_/tests/imports.rs
+expression: import one/wibble
+---
+error: Unknown module
+  ┌─ /src/one/two.gleam:1:1
+  │
+1 │ import one/wibble
+  │ ^^^^^^^^^^^^^^^^^
+
+No module has been found with the name `one/wibble`.
+Hint: Did you mean `import one.{wibble}`?
```

Comment:
Can you add a test like this but with a module name that's longer than 80 characters please.